sotodlib/site_pipeline/update_obsdb.py

```diff
@@ -0,0 +1,160 @@
+"""update_obsdb
+The config file could be of the form:
+.. code-block:: yaml
```

Comment:
I ended up needing to expand this a bit to work with the obsdb + obsfiledb together. Could you expand this example to this:

```
obsdb: obsdb.sqlite

obsfiledb: obsfiledb.sqlite
root_path: '/'

extra_files: ['M_index.yaml', 'M_book.yaml']
extra_extra_files: ['Z_bookbinder_log.txt']

cols:
  start_time: float
  stop_time: float
  n_samples: int
  telescope: str
  tube_slot: str
  type: str
  subtype: str
```